react/features/billing-counter/functions.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+
+import { jitsiLocalStorage } from '@jitsi/js-utils';
+
+import { BILLING_ID, VPAAS_TENANT_PREFIX } from './constants';
+import logger from './logger';
+
+/**
+ * Returns the full vpaas tenant if available, given a path.
+ *
+ * @param {string} path - The meeting url path.
+ * @returns {string}
+ */
+export function extractVpaasTenantFromPath(path: string) {
+    const [ , tenant ] = path.split('/');
+
+    if (tenant.startsWith(VPAAS_TENANT_PREFIX)) {
```

Comment:
So all meetings for vpass will have a tenant which starts with 'vpass-magic-cookie-'?

Reply:
Yes, apparently this is by design. The tenant will be `vpass-magic-cookie-{uuid}`.